snap/snapcraft.yaml

```diff
@@ -70,28 +109,33 @@ parts:
       - libreadline-dev
       - generate-ninja
 
-  lighting:
-    after: [connectedhomeip]
-    plugin: dump
-    source: .
+  local-bin:
+    plugin: nil
+    source: snap/local/bin/
     override-build: |
-      mkdir -p $CRAFT_PART_INSTALL/bin $CRAFT_PART_INSTALL/etc
-      cp run.sh $CRAFT_PART_INSTALL/bin/
-      cp lighting.py $CRAFT_PART_INSTALL/bin/
-      
-    stage-packages:
-      - python3
-      - python3-venv
+      mkdir -p $CRAFT_PART_INSTALL/bin
+      cp -v run.sh $CRAFT_PART_INSTALL/bin/
+      cp -v load-snap-options.sh $CRAFT_PART_INSTALL/bin/
 
 apps:
   lighting:
     daemon: simple
+    command-chain:
+      - bin/load-snap-options.sh
+    command: bin/run.sh
     install-mode: disable
     restart-delay: 10s
-    command: bin/run.sh
     plugs:
       - network
       - network-bind
       - bluez
       - avahi-control
-      - gpio # not connected automatically
+      - gpio
+  
+  # This app is to test the GPIO control without using a Matter controller
+  test-blink:
```

Comment:
```
ubuntu@ubuntu:~$ sudo snap start matter-pi-gpio-commander.test-blink
error: snap "matter-pi-gpio-commander" has no service "test-blink"
```

Reply:
The error message is clear. The snap has no such "service".

You can either run the app with `sudo snap start matter-pi-gpio-commander.test-blink` or `matter-pi-gpio-commander.test-blink`. If this isn't clear in the README, then we should fix that.

Reply:
Got it. This [commit](https://github.com/MonicaisHer/matter-pi-gpio-commander/commit/20fd4808231e94883ae8999cfcecb0bdd1b0e098) already fixed it.

Reply:
This one: https://github.com/MonicaisHer/matter-pi-gpio-commander/pull/3/commits/3cc1e92be4e3d8fe2b22d77ba285b25115821c22